Frontend/CO.CDP.OrganisationApp/Pages/Organisation/SupplierToBuyerOrganisationType.cshtml.cs

```diff
@@ -38,7 +39,7 @@ public IActionResult OnGet()
         if (!string.IsNullOrEmpty(BuyerOrganisationType) && !BuyerTypes.Keys.Contains(BuyerOrganisationType))
         {
             OtherValue = BuyerOrganisationType;
-            BuyerOrganisationType = "Other";
+            BuyerOrganisationType = StaticTextResource.SupplierToBuyer_OrganisationType_Other;
```

Comment:
BuyerTypes need translating on line 69

Reply:
ah I think the line numbers moved around. That wasn't the bit I was talking about 😆  Sorry, I should have been clearer.

I was referring to this:

```
public static Dictionary<string, string> BuyerTypes => new()
    {
        { "CentralGovernment", "Central government, public authority: UK, Scottish, Welsh and Northern Irish Executive"},
        { "RegionalAndLocalGovernment", "Regional and local government, public authority: UK, Scottish, Welsh and Northern Irish"},
        { "PublicUndertaking", "Public undertaking"},
        { "PrivateUtility", "Private utility"}
    };
```

But yeah, I guess that word "Other" also needed translating, so that's good.